lua/astrocommunity/pack/rust/init.lua

```diff
@@ -19,6 +19,12 @@ return {
       },
     },
   },
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts)
+      opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, { "rust_analyzer" })
+    end,
+  },
```

Comment:
rustup is the prefered way to handle rust_analyzer installations. This will be a no-go